samples/nrf9160/gnss/README.rst

```diff
@@ -93,6 +93,18 @@ CONFIG_GNSS_SAMPLE_ANTENNA_EXTERNAL - To use an external GNSS antenna
 CONFIG_GNSS_SAMPLE_ASSISTANCE_NRF_CLOUD - To use nRF Cloud A-GPS
    This configuration option enables A-GPS usage.
 
+.. _CONFIG_GNSS_SAMPLE_AGPS_FILTERED:
+
+CONFIG_GNSS_SAMPLE_AGPS_FILTERED - To limit to visible satellites
+   This is available when :kconfig:`CONFIG_GNSS_SAMPLE_ASSISTANCE_NRF_CLOUD` is enabled.
+   It reduces the ephemerides to only satellites that are at or above the :kconfig:`CONFIG_GNSS_SAMPLE_ELEVATION_MASK` angle right now.
+
+.. _CONFIG_GNSS_SAMPLE_ELEVATION_MASK:
+
+CONFIG_GNSS_SAMPLE_ELEVATION_MASK - To set satellite angle above horizon
+   This is available when :kconfig:`CONFIG_GNSS_SAMPLE_ASSISTANCE_NRF_CLOUD` is enabled.
```

Comment:
```suggestion
   This is available when :ref:`CONFIG_GNSS_SAMPLE_ASSISTANCE_NRF_CLOUD` is enabled.
```

Reply:
Same as the other edit.  Section not needed.